### PR TITLE
tests, multus: Add params to createBridgeNetworkAttachmentDefinition

### PR DESCRIPTION
**Pull request by orelmisan**
**What this PR does / why we need it**:
This allows creating a bridge Network Attachment Definition with
variable "type", "bridge" (name) and "macspoofchk" fields.

The change is required in order to test the MAC spoof check feature which is
currently available only with "cnv-bridge" (deployed by CNAO).

This PR is required for PR kubevirt/kubevirt#6766

Signed-off-by: Orel Misan <omisan@redhat.com>

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/network/vmi_multus.go [hunk 1]
```diff
@@ -382,9 +388,15 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 			}
 
 			table.DescribeTable("should be able to ping between two vms", func(interfaces []v1.Interface, networks []v1.Network, ifaceName, staticIPVm1, staticIPVm2 string) {
+				const (
+					bridgeCNIType         = "bridge"
+					bridgeName            = "br12"
```
Comment (EdDev) [inline tests/network/vmi_multus.go:393]: Why did you change the bridge name?

File: tests/network/vmi_multus.go [hunk 2]
```diff
@@ -170,8 +170,14 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 		nodes = util.GetAllSchedulableNodes(virtClient)
 		Expect(len(nodes.Items) > 0).To(BeTrue())
 
-		const vlanId = 100
-		Expect(createBridgeNetworkAttachmentDefinition(util.NamespaceTestDefault, linuxBridgeVlan100Network, vlanId, "")).To(Succeed())
+		const (
+			vlanId                = 100
+			bridgeCNIType         = "bridge"
+			bridgeName            = "br10"
+			macSpoofCheckDisabled = false
```
Comment (EdDev) [inline tests/network/vmi_multus.go:177]: I think these need to appear in a place where it can be used by both usages.
And the name needs to be fixed to the value, I mean:
```
vlanID100 = 100
bridge10 = "br10"
```

**Review by EdDev — CHANGES_REQUESTED** [review]

File: tests/network/vmi_multus.go (hunk 1 above)
Comment (orelmisan) [reply tests/network/vmi_multus.go:393]: Reverted back to "br10"

File: tests/network/vmi_multus.go (hunk 2 above)
Comment (orelmisan) [reply tests/network/vmi_multus.go:177]: Done

**Review by EdDev — APPROVED** [review]
Thank you!
